ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```
@@ -297,6 +297,8 @@ tests:
   run_if_changed: ^(.*/day2/.*|.*/cluster/.*)$
   steps:
     cluster_profile: packet-assisted
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:ocp-4-11
     env:
       ASSISTED_CONFIG: |
         NUM_MASTERS=1
```

An alternative that uses more stable 4.11 releases and have less affect on other components (see explanation on other comment):
```suggestion
    env:
      ASSISTED_CONFIG: |
        OPENSHIFT_VERSION=4.11
        NUM_MASTERS=1
```